qa/rpc-tests/disablewallet.py

```diff
@@ -10,6 +10,7 @@
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import *
 
+
```

Comment:
Not sure where this came from. Still, PEP suggests two empty lines, so I will just leave it as is.